src/components/Header/Header.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { useContext } from 'react';
+import { HeaderBar, HeaderDiv, BlankDiv, DMLogin } from './Header.styled';
+
+import { MenuBtn } from './Button/Menu/MenuBtn.jsx';
+import { MySearch } from './Input/Search/SearchField';
+import { MyDarkMode } from './Button/DarkMode/DarKMode';
+import { ThemeContext } from '../../providers/Theme/Theme.provider';
+import { Login } from './Button/Login/Login.jsx';
+
+function Header() {
+  const theme = useContext(ThemeContext);
+  const { darkMode } = theme.state;
+
+  return (
+    <HeaderBar className="Header" theme={darkMode}>
+      <HeaderDiv className="HeaderDiv">
+        <MenuBtn></MenuBtn>

```

Comment:
It is a good practice to use self closing tags when there is no children for a component
```suggestion
        <MenuBtn/>

```